### DISCUSSION TITLE
Getting an exception: "NotImplementedException: PropVariant VT_EMPTY".

### DISCUSSION THREAD

**Discussion by dhinuthampi** [naudio/NAudio#1071] [Q&A]
I am using the NAudio library to track audio. When connecting or disconnecting a headset or Bluetooth device, sometimes I am getting an exception: "NotImplementedException: PropVariant VT_EMPTY". I am using the MMNotificationClient interface to detect the change in the default audio device. Is this interface causing this exception, or what is the reason for getting this exception and how will it be fixed? Will using these interfaces or libraries result in this error?

**Comment by markheath** [comment]
I only implemented the PropVariant types I could reproduce. But I'm pretty sure VT_EMPTY just means null, so I could add that for the next version.

**Reply by dhinuthampi** [reply]
When will we get the new release? Is there a chance to get the NAudio library with these changes?
